### PR TITLE
File README was edited

### PR DESCRIPTION
**Pull request by Katy-dev**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,191 +1,90 @@
...
 
-[![@lhalam](https://avatars3.githubusercontent.com/u/3837059?s=100&v=4)](https://github.com/lhalam)
-[![@lhalam](https://avatars3.githubusercontent.com/u/3837059?s=100&v=4)](https://github.com/lhalam)
-[![@lhalam](https://avatars3.githubusercontent.com/u/3837059?s=100&v=4)](https://github.com/lhalam)
-[![@lhalam](https://avatars3.githubusercontent.com/u/3837059?s=100&v=4)](https://github.com/lhalam) 
-[![@lhalam](https://avatars3.githubusercontent.com/u/3837059?s=100&v=4)](https://github.com/lhalam)
-[![@lhalam](https://avatars3.githubusercontent.com/u/3837059?s=100&v=4)](https://github.com/lhalam)
-[![@lhalam](https://avatars3.githubusercontent.com/u/3837059?s=100&v=4)](https://github.com/lhalam)
-[![@lhalam](https://avatars3.githubusercontent.com/u/3837059?s=100&v=4)](https://github.com/lhalam)  
+[![@team](https://avatars2.githubusercontent.com/u/64921184?s=200&v=4)
```
Comment (LarysaShashuk) [inline README.md]: тут не закрита квадратна дужка (у всіх посиланнях на фотки)

Comment (Rominos7) [inline README.md]: Please remove square bracket

Comment (LarysaShashuk) [reply README.md]: рядок 72-76

File: README.md [hunk 2]
```diff
@@ -1,191 +1,90 @@
...
-> ideally one sentence
 
-> include terms/tags that can be searched
+ This project contains 4 main parts: client frontend, backend and database. All parts of the project are at https://github.com/ita-social-projects/YIF_Frontend
 
-**Badges will go here**
+Let's take a closer look at the technologies of each part.
 
-- build status
-- coverage
```
Comment (Romanarkko) [inline README.md:16]: Our covarage 90%

**Review by Romanarkko — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (Katy-dev) [reply README.md]: I have already fixed

Comment (Katy-dev) [reply README.md]: I have already fixed

File: README.md (hunk 2 above)
Comment (Katy-dev) [reply README.md:16]: I have already fixed

**Review by LarysaShashuk — APPROVED** [review]
approved

**Review by Rominos7 — APPROVED** [review]
Good

**Review by Romanarkko — APPROVED** [review]

**Pull request merged**